swaywm/sway
# Fix bottom border when using hide_edge_borders

If the user had a swaybar docked at the bottom, the view's bottom border would always be shown. The check needs to be based on the workspace height rather than the output height.

To test:

* Have `hide_edge_borders both` in config, a swaybar with `position bottom`, then start sway and open a view
* Repeat the above but with the swaybar set to `position top` to ensure I didn't break anything there
